jsonrpsee/src/lib.rs

```diff
@@ -79,7 +82,13 @@ pub use jsonrpsee_types as types;
 #[cfg(any(feature = "http-server", feature = "ws-server"))]
 pub use jsonrpsee_core::server::rpc_module::{RpcModule, SubscriptionSink};
 
-#[cfg(any(feature = "http-server", feature = "ws-server", feature = "core-client"))]
+#[cfg(any(
+	feature = "http-server",
+	feature = "ws-server",
+	feature = "async-client",
```

Comment:
do we need `feature = "client"` as well?

Reply:
I don't think so but added it too the be on the safe side :)